docker/thrift/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# global args
+ARG SPARK_VERSION=2.4.5
+ARG HADOOP_VERSION=2.7.7
+ARG HADOOP_MINOR_VERSION=2.7
+ARG HADOOP_HOME=/usr/local/hdp
+ARG SPARK_NAME=spark-${SPARK_VERSION}-bin-hadoop${HADOOP_MINOR_VERSION}
+ARG SPARK_HOME=/usr/local/spark
```

Comment:
Just curious - Do we need `SPARK_HOME` as an arg or would a simple `ENV` do the trick? 
(I'm not sure what the use case for having this as an ARG would be.)

Reply:
Thanks for the feedback here, I removed some of those ARGs, also to clean up the usage of ARG and ENV I re-use the base image

This seems to be a trick in Docker to propagate the environment but I'm happy with how it simplified the ARG/ENV usage in the Dockerfile

https://github.com/moby/moby/issues/37345#issuecomment-400250849